extensions/crossmodel-lang/src/glsp-server/mapping-diagram/handler/create-edge-operation-handler.ts

```diff
@@ -19,7 +19,7 @@ import { MappingModelState } from '../model/mapping-model-state.js';
 
 @injectable()
 export class MappingEdgeCreationOperationHandler extends JsonCreateEdgeOperationHandler {
-   @inject(ModelState) protected override modelState!: MappingModelState;
+   @inject(ModelState) protected declare modelState: MappingModelState;
```

Comment:
This change has been made in a number of files. It doesn't look quite correct to me: `declare` is usually reserved for declaration files or phantom fields, but `modelState`  is definitely supposed to be a real entity here. What is the reason for the change?

Reply:
I think I introduced this at some point since otherwise we got errors that the field was not initialized unless we added the definite assignment assertion (`!`) of which I'm not the biggest fan. I understand the argument that `declare` is also not the best way. Curiously, building and linting the application now, I don't see the error anymore, even without the `!`, so we could restore those statements.